opencost.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: opencost
+  version: 1.114.0
+  epoch: 0
+  description: Simple, fast container image builder for Go applications.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 47c4b6817505df7cf6d3c7f21bf318f0c80fd94e
+      repository: https://github.com/opencost/opencost
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X github.com/opencost/opencost/core/pkg/version.Version=${{package.version}}
+        -X github.com/opencost/opencost/core/pkg/version.GitCommit=$(git rev-parse HEAD)
+      output: costmodel
```

Comment:
should we consider name the binary `app`?
```bash
$ crane config ghcr.io/opencost/opencost | jq .config.Entrypoint
[
  "/go/bin/app"
]
```
I understand that we've a symlink but thinking if we can call this binary `app` as well. 💭

Reply:
gotcha, let's do that.